src/Space.cpp

```diff
@@ -108,6 +108,8 @@ Space::Space(Game *game, RefCountedPtr<Galaxy> galaxy, const SystemPath &path, S
 	GenSectorCache(galaxy, &path);
 }
 
+static void RelocateStarportIfNecessary(SystemBody *sbody, Planet *planet, vector3d &pos, matrix3x3d &rot, const std::vector<vector3d> &prevPositions);
```

Comment:
Style point, please move this to the top of the file so that all external function declarations are grouped up there, or include it's definition if it's already exposed somewhere else.

Reply:
This function is defined in the same file, just below, I didn't want to move it, now I just moved the whole function to the beginning of the file, will it go?